### ISSUE TITLE
pulseaudio: Daemon already running

### ISSUE THREAD

**Issue by eadwu** [NixOS/nixpkgs#44854]
## Issue description
The standard informative message of
```
E: [pulseaudio] pid.c: Daemon already running.
E: [pulseaudio] main.c: pa_pid_file_create() failed.
```
Doesn't seem to be a problem if `pulseaudio --start` (https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/display-managers/default.nix#L60) is run after `pulseaudio --daemonize=no` (https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/config/pulseaudio.nix#L252) though. Pulseaudio also works fine with just the user service and without `pulseaudio --start`.

### Steps to reproduce
```nix
{
  hardware = {
    pulseaudio = {
      configFile = pkgs.writeText "default.pa" (import /etc/nixos/hardware/pulseaudio/default.pa.nix { });
      enable = true;
      package = pkgs.pulseaudioFull;
      support32Bit = true;
    };
  };
}

...

      displayManager = {
        lightdm = {
          enable = true;

          greeters = {
            gtk = {
              clock-format = "%I:%M %p";
              enable = true;

              iconTheme = {
                name = "Papirus-Adapta-Nokto";
                package = pkgs.papirus-icon-theme;
              };

              theme = {
                name = "Adapta-Nokto";
                package = pkgs.adapta-gtk-theme;
              };
            };
          };
        };
      };

```

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.18.0-rc8, NixOS, 18.09.git.c2fe3b4 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1pre6338_45bcf541`
 - channels(root): `"nixos-18.09pre147850.2428f5dda13"`
warning: Nix search path entry '/home/xps/.nix-defexpr/channels' does not exist, ignoring
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by jtojnar** [comment]
> Pulseaudio also works fine with just the user service and without `pulseaudio --start`.

PulseAudio is socket activated nowadays so the line should be removed.

**Comment by eadwu** [comment]
Merged at b4d15afe0225c26c0cdeb4a6d9c662de827f6ae3.
